timesketch/lib/analyzers/utils.py

```diff
@@ -20,6 +20,143 @@
 except ImportError:
     from urllib import parse as urlparse  # pylint: disable=no-name-in-module
 
+# CDN domain list based on:
+# https://github.com/WPO-Foundation/webpagetest/blob/master/agent/wpthook/cdn.h
+# Last updated: 2019-01-11
+KNOWN_CDN_DOMAINS = {
+    '.att-dsa.net': 'AT&T',
+    '.pix-cdn.org': 'Advanced Hosters CDN',
+    '.akamai.net': 'Akamai',
+    '.akamaiedge.net': 'Akamai',
+    '.akamaihd.net': 'Akamai',
+    '.akamaitechnologies.com': 'Akamai',
+    '.akamaitechnologies.fr': 'Akamai',
+    '.akamaized.net': 'Akamai',
+    '.edgekey.net': 'Akamai',
+    '.edgesuite.net': 'Akamai',
+    '.srip.net': 'Akamai',
+    '.tl88.net': 'Akamai China CDN',
+    '.gslb.tbcache.com': 'Alimama',
+    '.cloudfront.net': 'Amazon CloudFront',
+    '.aads-cn.net': 'Aryaka',
+    '.aads-cng.net': 'Aryaka',
+    '.aads1.net': 'Aryaka',
+    '.azion.net': 'Azion',
+    '.azioncdn.com': 'Azion',
+    '.azioncdn.net': 'Azion',
+    'bo.lt': 'BO.LT',
+    '.bisongrid.net': 'Bison Grid',
+    '.bitgravity.com': 'BitGravity',
+    '.bluehatnetwork.com': 'Blue Hat Network',
+    '.b-cdn.net': 'BunnyCDN',
+    '.cdn77.net': 'CDN77',
+    '.cdn77.org': 'CDN77',
+    '.cdngc.net': 'CDNetworks',
+    '.gccdn.net': 'CDNetworks',
+    '.panthercdn.com': 'CDNetworks',
+    '.cdnsun.net': 'CDNsun',
+    '.cdnvideo.net': 'CDNvideo',
+    '.cdnvideo.ru': 'CDNvideo',
+    '.cachefly.net': 'Cachefly',
+    '.caspowa.com': 'Caspowa',
+    '.cedexis.net': 'Cedexis',
+    '.ccgslb.com': 'ChinaCache',
+    '.lxdns.com': 'ChinaNetCenter',
+    '.ourwebpic.com': 'ChinaNetCenter',
+    '.wscdns.com': 'ChinaNetCenter',
+    '.wscloudcdn.com': 'ChinaNetCenter',
+    '.cloudflare.com': 'Cloudflare',
+    '.cotcdn.net': 'Cotendo CDN',
+    '.systemcdn.net': 'Edgecast',
+    '.transactcdn.net': 'Edgecast',
+    '.v1cdn.net': 'Edgecast',
+    '.v2cdn.net': 'Edgecast',
+    '.v3cdn.net': 'Edgecast',
+    '.v4cdn.net': 'Edgecast',
+    '.v5cdn.net': 'Edgecast',
+    'edgecastcdn.net': 'Edgecast',
+    '.cdninstagram.com': 'Facebook',
+    '.fbcdn.net': 'Facebook',
+    '.fastly.net': 'Fastly',
+    '.fastlylb.net': 'Fastly',
+    '.nocookie.net': 'Fastly',
+    '.cdn.gocache.net': 'GoCache',
+    '.doubleclick.net': 'Google',
+    '.googleusercontent.com': 'Google',
+    '.gstatic.com': 'Google',
+    'googlehosted.com': 'Google',
```

Comment:
there are few of these domains... that is that don't start with a dot, shouldn't they all start with a dot?

in your check function you would then match on 

evilgooglehosted.com

That would be flagged as a CDN network... whereas I think you are just looking for goodie.googlehosted.com

Reply:
Good catch. Yeah I cleaned up the data and added dots.